src/gc-stacks.c

```diff
@@ -202,6 +202,10 @@ JL_DLLEXPORT void *jl_malloc_stack(size_t *bufsz, jl_task_t *owner) JL_NOTSAFEPO
     return stk;
 }
 
+extern _Atomic(int) gc_ptls_sweep_idx;
+extern _Atomic(int) gc_n_threads_sweeping;
+extern _Atomic(int) gc_stack_free_idx;
```

Comment:
Perhaps move these to `gc-stock.h` and include the header here? (`gc_n_threads_sweeping` is already defined there).